phoenix-core/src/it/java/org/apache/phoenix/end2end/CursorWithRowValueConstructorIT.java

```diff
@@ -669,4 +672,94 @@ public void testCursorsWithColsOfTypesTinyintSmallintFloatDouble() throws Except
             conn.close();
         }
     }
+
+    @Test
+    public void testCursorWithIndex() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        Connection conn = DriverManager.getConnection(getUrl(), props);
+
+        Statement stmt = conn.createStatement();
+
+        stmt.execute("DROP TABLE IF EXISTS SOME_NUMBERS");
+        String createTable = "CREATE TABLE IF NOT EXISTS SOME_NUMBERS\n" +
+                "(  \n" +
+                "   ID                             VARCHAR    NOT NULL,\n" +
+                "   NAME                           VARCHAR    ,\n" +
+                "   ANOTHER_VALUE                  VARCHAR    ,\n" +
+                "   TRANSACTION_TIME               TIMESTAMP  ,\n" +
+                "   CONSTRAINT pk PRIMARY KEY(ID)\n" +
+                ") " +
+                "IMMUTABLE_STORAGE_SCHEME=ONE_CELL_PER_COLUMN,\n" +
+                "UPDATE_CACHE_FREQUENCY=900000,\n" +
```

Comment:
I believe that these options are not relevent.
It'd best to remove them for clarity.